apps/ledger-live-desktop/tests/specs/speculos/send.tx.spec.ts

```diff
@@ -1,48 +1,265 @@
 import { test } from "../../fixtures/common";
 import { Account } from "../../enum/Account";
+import { Fee } from "../../enum/Fee";
 import { Transaction } from "../../models/Transaction";
 import { specs } from "../../utils/speculos";
 import { addTmsLink } from "tests/utils/allureUtils";
 import { getDescription } from "../../utils/customJsonReporter";
 
-// ONLY TESTNET (SEND WILL BE APPROVED ON DEVICE)
-const transactions = [
-  //TODO: Reactivate when fees will be stable
-  //new Transaction(Account.tBTC_1, Account.tBTC_2, "0.00001", "medium"),
-  new Transaction(Account.sep_ETH_1, Account.sep_ETH_2.address, "0.00001", "medium"),
+const transaction = new Transaction(Account.sep_ETH_1, Account.sep_ETH_2, "0.00001", Fee.MEDIUM);
+
+const transactionInvalidAddress = new Transaction(
+  Account.ETH_1,
+  Account.BTC_1,
+  "0.00001",
+  Fee.MEDIUM,
+);
+
+const tokenTransactionInvalid = {
+  transaction: new Transaction(Account.ALGO_USDT_1, Account.ALGO_USDT_2, "0.1", Fee.MEDIUM),
+  expectedErrorMessage: "Recipient account has not opted in the selected ASA.",
+};
+
+const tokenTransactionValid = new Transaction(
+  Account.ALGO_USDT_1,
+  Account.ALGO_USDT_3,
+  "0.1",
+  Fee.MEDIUM,
+);
+
+const transactionInputValid = [
+  new Transaction(Account.ETH_1, Account.ETH_2, "send max", Fee.MEDIUM),
+  new Transaction(Account.ETH_1, Account.ETH_2, "0.00001", Fee.MEDIUM),
 ];
 
-//Warning 🚨: Test may fail due to the GetAppAndVersion issue - Jira: LIVE-12581
-for (const [i, transaction] of transactions.entries()) {
-  test.describe("Send Approve", () => {
+const transactionsInputsInvalid = [
+  {
+    transaction: new Transaction(Account.ETH_1, Account.ETH_2, "", Fee.MEDIUM),
+    expectedErrorMessage: null,
+  },
+  {
+    transaction: new Transaction(Account.ETH_1, Account.ETH_2, "0", Fee.MEDIUM),
+    expectedErrorMessage: null,
+  },
+  {
+    transaction: new Transaction(Account.XRP_1, Account.XRP_2, "1", Fee.MEDIUM),
+    expectedErrorMessage: "Recipient address is inactive. Send at least 10 XRP to activate it",
+  },
+  {
+    transaction: new Transaction(Account.DOT_1, Account.DOT_2, "1.2", Fee.MEDIUM),
+    expectedErrorMessage: "Balance cannot be below 1 DOT. Send max to empty account.",
+  },
+  {
+    transaction: new Transaction(Account.DOT_1, Account.DOT_3, "0.5", Fee.MEDIUM),
+    expectedErrorMessage: "Recipient address is inactive. Send at least 1 DOT to activate it",
+  },
+  {
+    transaction: new Transaction(Account.ETH_1, Account.ETH_2, "100", Fee.MEDIUM),
+    expectedErrorMessage: "Sorry, insufficient funds",
+  },
+];
+
+//Warning 🚨: Test may fail due to the GetAppAndVersion issue - Jira: LIVE-12581 or insufficient funds
+
+test.describe("Send from 1 account to another", () => {
```

Comment:
do we need two level wrapping here with test+describe and then test+use? maybe having just test would be enough?